app/utilities/schema.py

```diff
@@ -92,39 +84,11 @@ def _load_schema_from_name(schema_name, language_code):
     return QuestionnaireSchema(schema_json, language_code)
 
 
-def transform_form_type(form_type):
-    census_form_types = {
-        "H": "household",
-        "I": "individual",
-        "C": "communal_establishment",
-    }
-
-    return census_form_types[form_type]
-
-
-def transform_region_code(region_code_input):
-    return region_code_input.lower().replace("-", "_")
-
-
-def transform_survey(survey_input):
-    return survey_input.lower()
-
-
-def get_schema_name_from_census_params(survey, form_type, region_code):
-    try:
-        form_type_transformed = transform_form_type(form_type)
-    except KeyError:
-        raise ValueError(
-            "Invalid form_type parameter was specified. Must be one of `H`, `I`, `C`"
-        )
-
-    region_code_transformed = transform_region_code(region_code)
-    survey_transformed = transform_survey(survey)
+def load_schema_from_params(eq_id, form_type, language_code=None):
```

Comment:
Is this been added just to test?

Reply:
Yeah, only for testing purposes.

Reply:
The launching of schemas using `eq_id` and `form_type` needs to be implemented. From the Trello card:
> Short/Medium term v2 schema selection strategy is required. This will need re-implementing to use `eq_id` and `form_type`.